integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -666,6 +666,21 @@ protected void checkDataFileExists(Path dir) {
     }
   }
 
+  protected void checkTableLocation(Path dir) {
```

Comment:
this function couldn't reused by other functions, it has some assumptions like the dir is `/user/hive/external_db`, so I prefer to check it directly not using a separate fuction.